app/src/main/java/agency/five/codebase/android/movieapp/ui/component/UserScoreProgressBar.kt

```diff
@@ -0,0 +1,53 @@
+package agency.five.codebase.android.movieapp.ui.component
+
+import androidx.compose.foundation.layout.*
+import androidx.compose.material.CircularProgressIndicator
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+
+
+@Composable
+fun UserScoreProgressBar(
+    score: Float,
+    modifier: Modifier = Modifier
+) {
+    Box{
+        CircularProgressIndicator(
+            progress = score,
+            color = Color.Black,
+            modifier = modifier
+                .width(40.dp)
+                .height(40.dp),
+
+            )
+        Text(
+            text = (score * 10).toString(),
+            fontSize = 14.sp,
+            modifier = modifier
```

Comment:
Modifiers which are passed in are designed so that they should be used by a single layout node in the composable function. Do not reuse modifiers because it will lead to undesired behaviour (such as padding being applied multiple times). Instead use a new `Modifier` here and anywhere else where there are reused modifiers